src/track/track.cpp

```diff
@@ -724,6 +724,11 @@ int Track::getAnalyzerProgress() const {
     return load_atomic(m_analyzerProgress);
 }
 
+
+QString Track::getCueFilePath() const{
```

Comment:
what class do you suggest adding it to? it was recommended to me by daschuer in [his review](https://github.com/daschuer/mixxx/pull/33). each track has its own cue file, so i thought it would be fitting information for the track class.

Reply:
You don't need a class, use a free function instead. Hide it in an anonymous namespace if it is only used in the implementation.